src/Illuminate/Auth/Access/Gate.php

```diff
@@ -100,20 +100,25 @@ public function has($ability)
      *
      * @param  string  $ability
      * @param  callable|string  $callback
+     * @param  array  $options
      * @return $this
      *
      * @throws \InvalidArgumentException
      */
-    public function define($ability, $callback)
+    public function define($ability, $callback, array $options = [])
     {
+        $options += ['guestable' => false];
```

Comment:
Better yet, you can skip this completely if you do a check using `Arr::get($options, 'guestable', false)` in your code